draft-ietf-webpush-protocol.xml

```diff
@@ -465,6 +465,9 @@ Location: https://push.example.net/message/qDIYHNcfAIPP_5ITvURr-d6BGt
           from the push service when a push message is delivered and then acknowledged
           by the user agent.
        </t>
+       <t>
+          A push service MUST support the processing of push message receipts.

```

Comment:
Since we have open issues with receipts - can we not make it mandatory ?

It's very unlikely all implementations will be able to support the receipts
any time soon,
 at least not as push promises.

Costin

On Fri, Jun 3, 2016 at 3:49 PM Brian Raymor <notifications@github.com>
wrote:

> In draft-ietf-webpush-protocol.xml
> <https://github.com/webpush-wg/webpush-protocol/pull/104#discussion_r65786758>
> :
>
> > @@ -465,6 +465,9 @@ Location: https://push.example.net/message/qDIYHNcfAIPP_5ITvURr-d6BGt
> >            from the push service when a push message is delivered and then acknowledged
> >            by the user agent.
> >         </t>
> > +       <t>
> > +          A push service MUST support the processing of push message receipts.
>
> What about a smaller change:
>
> *If the push service accepts the message for delivery, then it returns a
> 202 (Accepted) response.* A URI for the push message resource that was
> created in response to the request MUST be returned in the Location header
> field. The push service MUST also provide a URI for the receipt
> subscription resource in a link relation of type
> "urn:ietf:params:push:receipt".
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/webpush-wg/webpush-protocol/pull/104/files/2ed5a181f3ab889c0daefed762b4f22ad1f1c8de#r65786758>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAFI6hKqvL-Puf1eR8i4gDF_W1oEK7lrks5qIK-KgaJpZM4ItXyV>
> .
>